local-deployment/README.md

```diff
@@ -1,5 +1,9 @@
 This guide outlines how to deploy the explorer using a local lh-geth testnet. Utilized postgres, redis and little_bigtable as data storage
 
+# convenience-script run.sh
+
+The `run.sh` script in this directory can be used to start and stop everything. Just run `./run.sh start` to start the local chain and the explorer, then browse http://localhost:8080 to see it in action. You can run `./run.sh sql` to explore the sql-database. Everything can be stopped and cleaned up with `./run.sh stop`.
+
```

Comment:
**Nitpick:** All other chapters start with a capital letter.
Furthermore, the location of this is odd. A new user might expect that `run.sh` does everything for them while in fact the user first has to install docker and so on. Maybe place this chapter above "Exit Validators"?
Feel free to resolve this if you think this is not important.

Reply:
fixed

Reply:
i think it should be on top since its the fastest way to get everything running without reading through the whole file

Reply:
ok i put it after exit validators :)